scripts/src/chartprreview/chartprreview.py

```diff
@@ -79,41 +88,50 @@ def match_checksum(category, organization, chart, version):
     generated_digest = generated_report["metadata"]["tool"]["digest"]
 
     if  submitted_digest != generated_digest:
-        print("Digest is not matching:", submitted_digest, generated_digest)
+        msg = f"Digest is not matching: {submitted_digest}, {generated_digest}"
+        write_error_log(directory, msg)
         sys.exit(1)
 
-def check_url(report_path):
+def check_url(directory, report_path):
     report = yaml.load(open(report_path), Loader=Loader)
     chart_url = report["metadata"]["tool"]['chart-uri']
 
     try:
         requests.head(chart_url)
     except requests.exceptions.InvalidSchema as err:
-        print("Invalid schema:", chart_url)
-        print(err)
+        msgs = []
+        msgs.append(f"Invalid schema: {chart_url}")
+        msgs.append(str(err))
+        write_error_log(directory, *msgs)
         sys.exit(1)
     except requests.exceptions.InvalidURL as err:
-        print("Invalid URL:", chart_url)
-        print(err)
+        msgs = []
+        msgs.append(f"Invalid URL: {chart_url}")
+        msgs.append(str(err))
+        write_error_log(directory, *msgs)
         sys.exit(1)
     except requests.exceptions.MissingSchema as err:
-        print("Missing schema in URL:", chart_url)
-        print(err)
+        msgs = []
+        msgs.append(f"Missing schema in URL: {chart_url}")
+        msgs.append(str(err))
+        write_error_log(directory, *msgs)
         sys.exit(1)
 
-def match_name_and_version(category, organization, chart, version):
+def match_name_and_version(directory, category, organization, chart, version):
     submitted_report_path = os.path.join("charts", category, organization, chart, version, "report.yaml")
     if os.path.exists(submitted_report_path):
         submitted_report = yaml.load(open(submitted_report_path), Loader=Loader)
         submitted_report_chart_name = submitted_report["metadata"]["chart"]["name"]
         submitted_report_chart_version = submitted_report["metadata"]["chart"]["version"]
 
         if submitted_report_chart_name != chart:
-            print("Chart name is not matching against the value in the directory structure:", submitted_report_chart_name, "vs.", chart)
+            msg = f"[ERROR] Chart name is not matching against the value in the directory structure: {submitted_report_chart_name} vs. {chart}"
```

Comment:
repeat of previous message? Make them the same.

Reply:
Done. 229bd4e5c3b304f1c759c7b5b8f51971defae155